airflow/models/dagbag.py

```diff
@@ -623,11 +623,12 @@ def _serialize_dag_capturing_errors(dag, session):
                 )
                 self.log.debug("Calling the DAG.bulk_sync_to_db method")
                 try:
+                    DAG.bulk_write_to_db(self.dags.values(), session=session)
```

Comment:
Why are we flipping this order? I feel like I remember it was intentionally this way, but I can't remember why.

Reply:
if the order needs to be preserved, then we'll have to come up with something else.  we could abbreviate the URI i suppose?  then we would not need the id...  wdyt?